test/Microsoft.AspNetCore.Razor.Evolution.Test/TestRazorSourceDocument.cs

```diff
@@ -14,25 +16,34 @@ private TestRazorSourceDocument(string content, Encoding encoding, string filena
         {
         }
 
-        public static RazorSourceDocument CreateResource(string path, Encoding encoding = null)
+        public static RazorSourceDocument CreateResource(string path, Encoding encoding = null, bool normalizeNewLines = false)
```

Comment:
I don't think we should do that because a lot of the parser tests call these create methods and we shouldn't be normalizing new lines for those tests.